### PR TITLE
Support new checksum ID sort orders introduced in dfe-analytics v1.11.5

### PR DESCRIPTION
**Pull request by stevenleggdfe**
Note this required a substantial rework because BigQuery has a 100MB limit for the data processed across all aggregate functions within an individual subquery. Calculating checksums with three different sort orders depending on order_column causes this limit to be breached if completed within the same subquery. To work around this each order is calculated in a separate subquery and then recombined.

### CODE REVIEW

**Review by asatwal — COMMENTED** [review]
It's probably my understanding of Dataform/BQ, but Is it not possible to build a dynamic query with the order_column value as a parameter, rather than having 3 separate queries which differ only slightly (ie the order_column value) ?

**Comment by stevenleggdfe** [comment]
> It's probably my understanding of Dataform/BQ, but Is it not possible to build a dynamic query with the order_column value as a parameter, rather than having 3 separate queries which differ only slightly (ie the order_column value) ?

Good thinking, I've just done this

**Comment by asatwal** [comment]
👍 LGTM - I'll let someone who knows `js` better than me approve this PR.

File: includes/entity_ids_do_not_match.js [hunk 1]
```diff
@@ -1,4 +1,38 @@
 module.exports = (params) => {
+  function tablesWithMetricsSql(sortFields, ctx) {
+    return sortFields.map(sortField => {
+    return `tables_with_${sortField}_metrics AS (
+          SELECT
+            check.entity_table_name,
+            check.row_count AS database_row_count,
+            COUNT(DISTINCT entity_version.entity_id) AS bigquery_row_count,
```
Comment (stevenleggdfe) [inline includes/entity_ids_do_not_match.js:8]: Note this line has changed from counting updated_at to entity_id as a fix

**Review by ericaporter — APPROVED** [review]
Looks good!

**Pull request merged**